TestAssets/TestProjects/StandaloneTestApp/Program.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace StandaloneApp
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Hello, World!");
```

Comment:
oversight, will fix while working on parsing the output. 1 passing, 1 failing

---
In reply to: [66346432](https://github.com/dotnet/core-setup/pull/97#discussion_r66346432) [](ancestors = 66346432)